### DISCUSSION TITLE
configuration step by step for a newby

### DISCUSSION THREAD

**Discussion by creisey** [jcorporation/myMPD#1015] [Q&A]
hi there

i have lately used ympd and it was running fine for several years.

now with the latest updates it got slow and interrupted somehow with mpd.

id like to have it running on port 80, super simple without any special effects.
and like to have it started on boot

i have installed it with the debian prebuilt package here:
https://jcorporation.github.io/myMPD/installation/prebuild-packages-debian

then i tried to make a inital config with `mympd -c`, but that failed.

then i was trying this: 
`systemd-run -p DynamicUser=yes -p StateDirectory=mympd -p CacheDirectory=mympd -E MYMPD_LOGLEVEL=4 -E MYMPD_HTTP_PORT=0 -E MYMPD_SSL_PORT=1333 mympd -c` without any success.
it created some deamon services but there was no webinterface.

and no config was created here: /var/lib/mympd/config/

would anyone be kind to help me?

**Comment by jcorporation** [comment]
Always post logs and error messages. Read the documentation to set the environment variables correctly. Or easier: start myMPD once, stop it and change the configuration files afterwards.

P.S.
Your commandline disables http and enables https on port 1333.

**Comment by creisey** [comment]
well here's what i did:
`sudo systemd-run -p DynamicUser=yes -p StateDirectory=mympd -p CacheDirectory=mympd -E MYMPD_HTTP_PORT=80 mympd -c`

output:
Running as unit: run-r1bfe7b54c0504d429a0773952d5b8fc5.service

```
pi@superc:~ $ sudo systemctl status mympd
● mympd.service - myMPD server daemon
   Loaded: loaded (/usr/lib/systemd/system/mympd.service; disabled; vendor preset: enabled)
   Active: failed (Result: signal) since Thu 2023-04-13 16:30:36 UTC; 29s ago
     Docs: man:mympd(1)
  Process: 1314 ExecStart=/usr/bin/mympd (code=killed, signal=SEGV)
 Main PID: 1314 (code=killed, signal=SEGV)
```
/var/lib/mympd/config/ 
stays empty

**Comment by jcorporation** [comment]
Which Debian version? Output of `journalctl -u mympd`

**Reply by creisey** [reply]
raspian buster = debian 10

**Reply by jcorporation** [reply]
Any output for the journalctl command?

Only systemd v235 and above is supported.
